util/fibers/dns_resolve.cc

```diff
@@ -151,8 +146,9 @@ void ProcessChannel(ares_channel channel, AresChannelState* state, DnsResolveCal
 
 }  // namespace
 
-error_code DnsResolve(string host, uint32_t wait_ms, char dest_ip[], ProactorBase* proactor) {
+error_code DnsResolve(string host, uint32_t wait_ms, absl::Span<char> dest_ip, ProactorBase* proactor) {
```

Comment:
I prefer keeping the type as `char dest[]` and update the comments around this function

Reply:
Changed. But why? Span is implicitly constructible from char arrays...

Reply:
I consider it an awkward type for public interfaces.